nixos/modules/services/networking/veilid.nix

```diff
@@ -0,0 +1,227 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.veilid;
+  dataDir = "/var/db/veilid-server";
```

Comment:
Why `/var/db` instead of `/var/lib`?

I'm not sure what the rules are but on my desktop, I only have 1 folder in db (`dhcpcd`), while I have 34 in lib.

Reply:
/var/db was referenced from the [config](https://gitlab.com/veilid/veilid/-/blob/main/package/linux/veilid-server.conf?ref_type=heads#L27) included in the rpm and deb packages and from the example [config](https://gitlab.com/veilid/veilid/-/blob/main/doc/config/veilid-dev-node-config?ref_type=heads#L36) in the documentation.
/var/lib is also fine.